test/http.js

```diff
@@ -174,6 +174,29 @@ describe('http', () => {
         url: 'https://swagger.io?one=1&two=2&three=3'
       })
     })
+
+    it.only('should not encode form-data', function () {
+      const FormData = require('isomorphic-form-data')
```

Comment:
The Node version of FormData doesn't expose standard methods like `get`, `keys`... Easiest way to test is monkey patching `append`... Not great, but quick and simple.